bpmnconverter/README.md

```diff
@@ -1,3 +1,27 @@
+# BPMN to Lucid converter
```

Comment:
It looks like most / all of this is a duplicate of what's already in this file. Would you remove this / move any actual additions into the text further down?